packages/odyssey-react-mui/src/Button.tsx

```diff
@@ -97,7 +96,7 @@ export type ButtonProps = {
   /**
    * The variant of the Button
    */
-  variant: (typeof buttonVariantValues)[number];
+  variant: (typeof buttonVariantValues)[number] | "tertiary";
```

Comment:
Curious why we're accepting `"tertiary"` as a variant option here, but removed it elsewhere?

My hunch is to make sure we don't have a breaking change, but I feel it's reasonable to remove this variant from code and the typings, in general, than give a false impression that it's still a variant if someone attempts to use it such way?

From a user's perspective, they may be unaware that their tertiary buttons are no longer supported, but there's no compile time warning or error to explain why their tertiary buttons like secondary ones.